src/psyclone/domain/lfric/lfric_stencil.py

```diff
@@ -0,0 +1,86 @@
+# -----------------------------------------------------------------------------
```

Comment:
Looking at this class, it seems to me we could do without it and have a NamedTuple instead:

    LFRicArgStencil = namedtuple("Symbol", "name extent extent_arg direction_arg")

I say this because none of the setters do any validation. Can @rupertford, @TeranIvy or @hiker see any potential problem with doing that? (e.g. is this class likely to need significant additional functionality in future?)

Reply:
@arporter, I think that the potential future functionality would be "more of the same" in the sense of more stencil types, but the underlying mechanism wouldn't change. If doing away with the class doesn't affect adding more stencil types then I'm fine with it!